server/pxf-json/src/test/resources/parser-tests/offset/input.json

```
@@ -0,0 +1,7 @@
+[
+{"cüsötmerstätüs":"välid","name": "äää", "year": "2022", "address": "söme city", "zip": "95051"},
```

```suggestion
{"cüstömerstätüs":"välid","name": "äää", "year": "2022", "address": "söme city", "zip": "95051"},
```
unless intentional. Here and everywhere else, seems like a typo